bgpq4.c

```diff
@@ -420,6 +428,8 @@ main(int argc, char* argv[])
 			} else if (expander.vendor == V_NOKIA ||
 			    expander.vendor == V_NOKIA_MD) {
 				expander.aswidth = 8;
+			} else if (expander.vendor == V_ARISTA) {
+				expander.aswidth = 4;
```

Comment:
This, and the `T_OASPATH` below are both borrowed from the `V_CISCO` options higher in the conditional.